packages/components/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 
 ### Enhancements
 
+-   Add `onClick` event on `FormFileUpload` to emit `onChange` event when selecting the same file again.
```

Comment:
We'll need to move this into the "Unreleased" section before merging, or else it will auto-merge into the wrong section. (I don't think this usually happens, but a lot of our changelog entries on open PRs got borked because of all the backed up PRs in the past two weeks due to CI problems!)

Reply:
Fixed in 911a417.